pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -106,9 +106,9 @@ def on_evaluation_model_train(self, *args, **kwargs):
 
     def on_evaluation_end(self, *args, **kwargs):
         if self.testing:
-            self.trainer.call_hook('on_test_end', *args, capture=True, **kwargs)
+            self.trainer.call_hook('on_test_end', *args, **kwargs)
         else:
-            self.trainer.call_hook('on_validation_end', *args, capture=True, **kwargs)
+            self.trainer.call_hook('on_validation_end', *args, **kwargs)
```

Comment:
could do:

```suggestion
        self.trainer.call_hook('on_test_end' if self.testing else 'on_validation_end', *args, **kwargs)
```

Reply:
I would prefer not. I found the current version simpler to understand for new users.